### PR TITLE
cirrus module: exposed some Starlib builtins and added an env dict

### PR DESCRIPTION
**Pull request by edigaryev**
See #53.

### CODE REVIEW

File: pkg/larker/loader/loader.go [hunk 1]
```diff
@@ -65,11 +71,28 @@ func (loader *Loader) LoadFunc() func(thread *starlark.Thread, module string) (s
...
 			return starlark.StringDict{
 				"fs": &starlarkstruct.Module{
 					Name:    "fs",
 					Members: builtin.FS(loader.ctx, loader.fs),
 				},
+				"env":  starlarkEnv,
+				"http": starlibModules["http"],
+				"hash": starlibModules["hash"],
+				"json": starlibModules["json"],
+				"yaml": starlibModules["yaml"],
```
Comment (fkorotkov) [inline pkg/larker/loader/loader.go:95]: Let's also add `zipfile`, `re` and `base64. Can do here or in a separate PR.

File: pkg/larker/loader/loader.go [hunk 2]
```diff
@@ -65,11 +71,28 @@ func (loader *Loader) LoadFunc() func(thread *starlark.Thread, module string) (s
...
+					return nil, err
+				}
+			}
+
 			return starlark.StringDict{
 				"fs": &starlarkstruct.Module{
 					Name:    "fs",
 					Members: builtin.FS(loader.ctx, loader.fs),
 				},
+				"env":  starlarkEnv,
```
Comment (fkorotkov) [inline pkg/larker/loader/loader.go:91]: Seems we can rename `starlibModules` to `builtinModules` and then do something like:

```python
builtinModules["fs"] = ...
builtinModules["env"] = ...
return builtinModules
```

This way there is no need for duplicated `"http"`, `"hash"`, etc. What do you think?

Comment (edigaryev) [reply pkg/larker/loader/loader.go]: See https://github.com/cirruslabs/cirrus-cli/pull/96/commits/9645b0411b06fbfe0062c77cc16d3f023fae8734.

File: pkg/larker/loader/loader.go (hunk 1 above)
Comment (edigaryev) [reply pkg/larker/loader/loader.go]: See https://github.com/cirruslabs/cirrus-cli/pull/96/commits/40d9ac23705bfac15525e21c0c14fae01b6909f8.

**Review by fkorotkov — APPROVED** [review]
